### ISSUE TITLE
Cannot read magnetic field Bz

### ISSUE THREAD

**Issue by MerlinPendragon** [skuschel/postpic#220]
Please tell us which version of postpic you are using by adding the output of
```python
import postpic as pp
print(pp.__version__)
v0.4+54.g66bc0f4
```
Please also add if you are using python2 or python3. You can find out using
```python
import sys
print(sys.version)
'3.6.6 |Anaconda custom (64-bit)| (default, Jun 28 2018, 17:14:51) \n[GCC 7.2.0]'
```
Hi guys,

It seems that every time I'm trying to use `dr.Bz()` to get magnetic field Bz, my jupyter lab kernel got stuck and need to be restarted. Could you please find out why? Other field variable like Ey works correctly.

Here is my sdf

[20005.sdf.zip](https://github.com/skuschel/postpic/files/3415368/20005.sdf.zip)

**Comment by Ablinne** [comment]
Thanks for the report. I have tried to reproduce the issue with your file with `postpic v0.4+54.g66bc0f4` on python 3.7.3, python 3.6.5 and python 3.6.6. In every case `dr.Bz()` worked just fine and returned the expected data.
To me this sounds like a problem with the low level `sdf` module which is provided by the Epoch devs. Please get a current version of the Epoch source code and rebuild and reinstall your `sdf` module by running `make` in the `SDF/C` directory and `./build` in the `SDF/utilities` directory of the epoch source.

**Comment by MerlinPendragon** [comment]
Thank you Ablinne! 

The problem is solved after upgrading `sdf` to v2.6.7.
